LocationSync/LocationSync.cpp

```diff
@@ -149,7 +174,12 @@ namespace Plugin {
             subSystem->Release();
 
             if ((_sink.Location() != nullptr) && (_sink.Location()->TimeZone().empty() == false)) {
-                Core::SystemInfo::SetEnvironment(_T("TZ"), _sink.Location()->TimeZone());
+                auto timeZone = readTextFile(TZ_FILE);
```

Comment:
Here you could evaluate the the config parameter again and only if it is not set, set the Core::SystemInfo::SetEnvironment(_T("TZ"), _sink.Location()->TimeZone());
Currently it might be a bit confusing. Seems if the file is, at the moment of running, empty, the timezone retrieved is set, if it was not empty, the set one is set. THis makes the functionality very time depending and thus hard to predict. Suggest to not set if the override is set to create a consistent behaviour over time.